src/utils/homeRunDistance.ts

```diff
@@ -0,0 +1,23 @@
+export function positionToHomeRunDistance(distance: number, units: "feet" | "meters" = "feet"): number {
+  // In NTSC vs. PAL, meters are different sizes
+  const feetModeConversionFactor = 0.952462;
+  const metersModeConversionFactor = 1.04167;
```

Comment:
```suggestion
  const FEET_CONVERSION_FACTOR = 0.952462;
  const METERS_CONVERSION_FACTOR = 1.04167;
```
Can we change these constants casing and move them out of the function and to the top of the file?